docker-compose/README.md

```diff
@@ -38,15 +40,15 @@ docker-compose up
 
 
 #### Other usefull commands
-* You can **run everythng in the background** by adding the "-d" argument to the "docker-compose up" command: ```docker-compose up -d```
-* You can **stop everything** with CTRL-C or ```docker-compose down```
+* You can **run everythng in the background** by adding the "-d" argument to the "docker compose up" command: ```docker compose up -d```
```

Comment:
A typo: `everythng`

Reply:
fixed.
also ran black for code formatting.